packages/atomic/src/components/search/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -18,14 +19,23 @@ type Regions = {[regionName: string]: {assertive: boolean; message: string}};
  */
 @Component({
   tag: 'atomic-aria-live',
+  styles: `
+    :host {
+      position: absolute;
+      display: block;
+      height: 0;
+      overflow: hidden;
+      margin: 0;
+    }
+  `,
   shadow: false,
 })
 export class AtomicAriaLive {
   @Element() private host!: HTMLAtomicAriaLiveElement;
   @State() private regions: Readonly<Regions> = {};
 
   private messagesQueue = buildDebouncedQueue({delay: 500});
-  private id = randomID('aria-live-');
+  private id!: string;
```

Comment:
Tokeep: No assignment on natively reflective attributes (so most if not all properties of elements) before connectedCallback (ergo: ctor :x: and declaration/assignment)